src/message/get-log/index.tsx

```diff
@@ -42,7 +38,7 @@ export const GetLog: React.FC<GetLogProps> = ({ station }) => {
       operation: 'ChargingStationSequencesGet',
       gqlQuery: CHARGING_STATION_SEQUENCES_GET_QUERY,
```

Comment:
you will need to adjust the query here to handle a list of charging stationIds as changed below, which means that the query will in theory also return a list of items... this becomes a problem, because we are using the response that we get from this query, down below, to set the requestId - ```const getLogRequest = new GetLogRequest();
  getLogRequest[GetLogRequestProps.requestId] =
    requestIdResponse?.data?.ChargingStationSequences[0]?.value ?? 0;```
    
    we may want to discuss with the team how this should be handled 🙏

Reply:
Changed the query to use a in instead of eq